1-Calling-MSGraph/1-1-AzureAD/Console-Interactive-MultiTarget/Program.cs

```diff
@@ -8,15 +8,18 @@
 
 namespace Console_Interactive_MultiTarget
 {
-    class Program
+    internal class Program
     {
         private static PublicClientApplicationOptions appConfiguration = null;
         private static IConfiguration configuration;
-        private static string _authority;
+        private static string MSGraphURL = "https://graph.microsoft.com/v1.0/";
```

Comment:
Could we add this to the configuration? (See the other samples, like the daemon one)

Reply:
Updated the code.
Thanks.